x-pack/plugins/spaces/server/saved_objects/mappings.ts

```diff
@@ -30,10 +30,10 @@ export const SpacesSavedObjectMappings = deepFreeze({
     disabledFeatures: {
       type: 'keyword',
     },
-    imageUrl: {
-      type: 'text',
-      index: false,
-    },
+    // imageUrl: {
```

Comment:
without `dynamic: false` at the top-level of the mappings this will mean documents can't be created if they have an imageUrl

Reply:
@rudolf,  the `SavedObjectsTypeMappingDefinition` doesn't allow `dynamic: true`:

``` 
// in packages/core/saved-objects/core-saved-objects-server/src/mapping_definition.ts

type 'SavedObjectsTypeMappingDefinition'.
--
  | proc [tsc]   Types of property 'dynamic' are incompatible.
  | proc [tsc]     Type 'true' is not assignable to type 'false \| "strict" \| undefined'
```

Reply:
@rudolf finally, a green build! Let's see if updating from main causes any issues.

Reply:
I've re-requested a review.